server/src/utils/fight/updateAchievements.ts

```diff
@@ -6,101 +7,46 @@ const updateAchievements = async (
   store: AchievementsStore,
   isTournamentFight: boolean,
 ) => {
+  const bruteAcheivments = [];
+  const bruteAcheivmentsTournamentRelated = [];
```

Comment:
```suggestion
  const bruteAchievments = [];
  const bruteAchievmentsTournamentRelated = [];
```

Reply:
I don't see any difference in your suggestion ^^"